### PR TITLE
zh-CN: update glossary/accent

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

bright color 明亮的颜色 => 靓丽的颜色

### Motivation

the bright may refer to "strong and easy to see", but not "has a very high brightness"

### Additional details

Oxford Modern Dictionary 10 example sentence:

I like bright colours => 我喜欢靓丽的颜色

### CODE REVIEW

File: files/zh-cn/glossary/accent/index.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
 title: 强调色
 slug: Glossary/Accent
+l10n:
+  sourceCommit: 7cec92742c6a79071626cb0fbfd0acde425e5814
```
Comment (yin1999) [inline files/zh-cn/glossary/accent/index.md:5]: The sourceCommit should refer the last commit of the corresponding file: https://github.com/mdn/content/commits/main/files/en-us/glossary/accent/index.md

So, fo this one: `ada5fa5ef15eadd44b549ecf906423b4a2092f34`

Comment (PassionPenguin) [reply files/zh-cn/glossary/accent/index.md:5]: hah…? not the filehash but the commit hash?😭

Comment (yin1999) [reply files/zh-cn/glossary/accent/index.md]: Note the name of this, "sourceCommit" which contains "commit"

Comment (yin1999) [reply files/zh-cn/glossary/accent/index.md]: > then those i commited and merged needs an update…

Plase open PRs for them. Thanks

**Comment by PassionPenguin** [comment]
ptal @yin1999

File: files/zh-cn/glossary/accent/index.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 title: 强调色
 slug: Glossary/Accent
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-**强调色**（accent）通常为一种明亮的颜色，与颜色方案中更实用的背景和前景色形成对比。这些颜色存在于很多平台（但不是所有平台）的视觉样式中。
+**强调色**（accent）通常为一种靓丽的颜色，与颜色方案中更实用的背景和前景色形成对比。这些颜色存在于很多平台（但不是所有平台）的视觉样式中。
```
Comment (yin1999) [inline files/zh-cn/glossary/accent/index.md:10]: 靓丽一般用于形容美丽，可能用在这里并不是很合适（这里特指颜色能够和周围的其他内容形成对比）。

Comment (PassionPenguin) [reply files/zh-cn/glossary/accent/index.md:10]: Yeah…望文生义了

What’bout 鲜明的色彩？

```suggestion
**强调色**（accent）通常为一种鲜明的颜色，与颜色方案中更实用的背景和前景色形成对比。这些颜色存在于很多平台（但不是所有平台）的视觉样式中。
```

**Review by yin1999 — APPROVED** [review]
感谢！

**Pull request merged**